src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -57,7 +57,7 @@ def _post(self, request: Request):
         if request.relay_request_data.get("globalConfig"):
             metrics.incr("relay.project_configs.global.fetched")
             global_config = get_global_config()
-            response["global"] = global_config
+            response["global_config"] = global_config
```

Comment:
@TBS1996 since the endpoint is to request configs, `global_config` is redundant and `global` should be good enough. We specified this decision in [this document (internal)](https://www.notion.so/sentry/Global-config-protocol-2aa9743c827648c0b577b835d344b6d8?pvs=4). Could we please revert this change?